docs/developer-guide/new-role.rst

```diff
@@ -0,0 +1,50 @@
+.. Copyright (C) 2021 Robin Schneider <ypid@riseup.net>
+.. Copyright (C) 2021 DebOps <https://debops.org/>
+.. SPDX-License-Identifier: GPL-3.0-or-later
+
+Creating a new DebOps role
+==========================
+
+Historically, a new role was created by copying an existing role and doing an
+initial search and replace. Which has the following problems:
+
+- Which role is up to the latest standards and thus suitable for creating the next role?
+- Requires manual work to setup a new role skeleton before work can be started (search and replace).
+- For new contributers, this is even more difficult.
+- The role standards are going to evolve thus the roles need to be updated.
+
+DebOps uses `Copier`__ to help address those issues.
+
+  .. __: https://copier.readthedocs.io/en/stable/
+
+Setup
+-----
+
+- Install copier.
+- Clone https://github.com/ypid/debops-template.git next to the debops monorepo (or symlink it next to it).
```

Comment:
Not sure if cloning is the right way to go here...
The role will probably not be kept in sync with the template repo, it's meant to be it's own thing

maybe give a git-archive command, or point to github's generated tarballs...

Reply:
Please read the Copier docs first ;-)